doc/example_xmpp_envs.conf

```diff
@@ -11,6 +11,9 @@
             // The base XMPP domain
             xmpp-domain = "xmpp-domain"
 
+            // An (optional) base url the Jibri will join if it is set
+            base-url = "http://meet.jit.si"
```

Comment:
Can you please leave it commented out so people don't enable it by mistake?

Also use https://meet.example.com to make it obvious it needs changing.

Reply:
done